### PR TITLE
Add webpack e2e example

### PR DESCRIPTION
**Pull request by mattlewis92**
I know are quite a lot of people that struggle to get things like this setup (myself included!) so I thought I'd add a full e2e example that shows how to add this to a webpack + karma + typescript project.

### CODE REVIEW

**Comment by paxos** [comment]
This is gold. Thank you!

**Review by robmcm — APPROVED** [review]
This would be really handy

**Comment by seeruk** [comment]
You could use a global git ignore file for `.idea/` and `.DS_Store`, then you can keep your personal development environment configuration out of the repository.

**Comment by mattlewis92** [comment]
@SeerUK TIL - thanks for the tip. Updated the PR with your suggestion :)

**Comment by mattlewis92** [comment]
@SeerUK TIL - thanks for the tip. Updated the PR with your suggestion :)

**Comment by mattlewis92** [comment]
@SeerUK TIL - thanks for the tip. Updated the PR with your suggestion :)

**Comment by mattlewis92** [comment]
@SeerUK TIL - thanks for the tip. Updated the PR with your suggestion :)

**Comment by robmcm** [comment]
Doesn't look there there is any action on these PRs though :(

**Comment by mattlewis92** [comment]
@delasteve @marcules I'm more than happy to take over maintenance of this module if you add me as a contributor 😄

**Comment by marcules** [comment]
Hi @mattlewis92 I just added you and @mateuszjuszkiewicz to the github repo and npm as collaborators.

I'm currently (and in the last few months) busy with IRL stuff (moving, job change) and not really able to contribute (or approve / review) any of this.

I can transfer this project to someone else if you guys want; I'll be in freenode#karma-remap-istanbul the next few days ~GMT2100-2400 to discuss this with anyone who want's to maintain it. (ping @delasteve )

Greetings,
Marc

File: examples/webpack/karma.conf.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+const webpackConfig = {
+  module: {
+    loaders: [{
+      test: /\.ts$/,
+      loader: 'ts',
+      exclude: /node_modules/
+    }],
+    postLoaders: [{
+      test: /src\/.+\.ts$/,
+      exclude: /(test|node_modules|\.spec)/,
```
Comment (mat-82) [inline examples/webpack/karma.conf.js]: Similar reg-ex bit us recently on a project that was called bundle-test. Took me a bit to find out why coverage was not recorded.

Comment (mattlewis92) [reply examples/webpack/karma.conf.js]: Good catch, I think that was left over in my copy pasting this from my projects config. How about `exclude: /(node_modules|\.spec\.ts$)/,`

**Comment by mattlewis92** [comment]
@marcules thanks! Best of luck with everything IRL 😄

File: examples/webpack/karma.conf.js (hunk 1 above)
Comment (mat-82) [reply examples/webpack/karma.conf.js]: I think that would work better. Do you have time to make that change, I think we can merge once this is done.

Comment (mattlewis92) [reply examples/webpack/karma.conf.js]: Yup I'll make the change tomorrow and then merge 👍

**Pull request merged**